curriculum/challenges/chinese/01-responsive-web-design/applied-visual-design/add-a-box-shadow-to-a-card-like-element.chinese.md

```diff
@@ -46,9 +67,9 @@ tests:
     text-align: left;
     color: black;
   }
-
-
-
+  
```

Comment:
Please remove those spaces.

Reply:
I think these spaces were supposed to remind the user to add a new style right here.